src/services/refactor.service.js

```diff
@@ -0,0 +1,9 @@
+export const refactorPhoneNumber = (phone = '') => {
```

Comment:
Please, rename this method to the `formatPhoneNumber`. because it makes refactoring

Reply:
fixed it